flow/designs/asap7/mock-array-big/Element/pdn.tcl

```diff
@@ -3,17 +3,24 @@
 ####################################
 add_global_connection -net {VDD} -inst_pattern {.*} -pin_pattern {^VDD$} -power
 add_global_connection -net {VSS} -inst_pattern {.*} -pin_pattern {^VSS$} -ground
+
 ####################################
 # voltage domains
 ####################################
 set_voltage_domain -name {CORE} -power {VDD} -ground {VSS}
+
 ####################################
 # standard cell grid
 ####################################
 define_pdn_grid -name {top} -voltage_domains {CORE}
-add_pdn_stripe -grid {top} -layer {M1} -width {0.018} -pitch {0.54} -offset {0} -followpins
-add_pdn_stripe -grid {top} -layer {M2} -width {0.018} -pitch {0.54} -offset {0} -followpins
-add_pdn_stripe -grid {top} -layer {M5} -width {0.12} -spacing {0.072} -pitch {11.88} -offset {0.300}
+add_pdn_ring    -grid {top} -layers {M5 M4} -widths {0.12 0.12} -spacings {0.072} -core_offset {0.084}
```

Comment:
add some comments here as to how these magic numbers where chosen and what requirements they satisfy.